### DISCUSSION TITLE
MCPHub: A Collection and Discovery Service for MCP Servers

### DISCUSSION THREAD

**Discussion by shenli** [modelcontextprotocol/.github#67] [Show and tell]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

Hi all! I've built a hub for discovering and accessing MCP servers.

- [MCP Hub](https://www.mcphub.ai)
Browse and search through a curated collection of MCP servers: https://www.mcphub.ai/servers. Find implementations by features, language, or environment

- MCPHub Server
It can also be used as a MCP server. You can search/view mcp servers in Claude desktop.
Setup guide: [mcphub.ai/quickstart](https://www.mcphub.ai/quick-start)

Would love to hear your thoughts and feedback! Let me know if you have a server to add to the collection or ideas for improving the discovery experience.

**Comment by VioletJet1728** [comment]
Love the branding/look and feel.
